llama/s3.py

```diff
@@ -1,28 +1,39 @@
+from boto3 import client
 from s3_concat import S3Concat
 
 
-def concatenate_files(date, session, bucket, key_prefix, output_filename):
-    """Concatenates files with the provided key_prefix in the provided bucket in a
-    random order"""
-    job = S3Concat(
-        bucket,
-        output_filename,
-        None,
-        session=session,
-    )
-    job.add_files(f"{key_prefix}{date}")
-    job.concat()
+class S3:
```

Comment:
Do we want this name to be more descriptive than a generic `S3`?

Reply:
I couldn't think of anything better, S3Client was confusing. Open to suggestions!